consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java

```diff
@@ -155,21 +155,21 @@ public void roundExpired(final RoundExpiry expire) {
 
   @Override
   public void handleProposalPayload(final Proposal proposal) {
-    LOG.debug("Received a Proposal Payload.");
+    LOG.trace("Received a Proposal Payload.");
```

Comment:
I think these receives would be useful for debugging

Reply:
these are in debug in the IbftRound (along with message sends) - thus I figured these were largely just a pass through (and potentially do not require logging at all).

Reply:
good point. there is no benefit in logging those at the height manager and in the ibft round. so happy for this to be trace and also for the other receives in the height manager.